EXILED/Exiled.Events/Patches/Events/Player/Shooting.cs

```diff
@@ -39,34 +34,59 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
             List<CodeInstruction> newInstructions = ListPool<CodeInstruction>.Pool.Get(instructions);
 
             Label returnLabel = generator.DefineLabel();
-
-            int offset = 1;
-            int index = newInstructions.FindIndex(instruction => instruction.opcode == OpCodes.Stloc_2) + offset;
+            Label continueLabel = generator.DefineLabel();
+
+            /*
+            [] <= Here
+            IL_0078: ldarg.2      // processingMethod
+            IL_0079: ldarg.0      // this
+            IL_007a: ldfld        class ReferenceHub InventorySystem.Items.Firearms.Modules.Misc.ShotBacktrackData::PrimaryTargetHub
+            IL_007f: callvirt     instance void class [mscorlib]System.Action`1<class ReferenceHub>::Invoke(!0/*class ReferenceHub* /)
+             */
+            int hasTargetIndex = newInstructions.FindIndex(instruction => instruction.IsLdarg(2));
+            List<Label> hasTargetLabels = newInstructions[hasTargetIndex].ExtractLabels();
```

Comment:
there is no labels on that instruction i suppose, why do you extract labels there?

Reply:
Yeah, right, but there are no labels on that instruction to change their, so you just extract 0 labels from that and place 0 labels to the new instruction😔

Reply:
Oh maybe i am a shizo and there are really labels i dont see💔